### PR TITLE
Filter component

### PR DESCRIPTION
**Pull request by RodrigoRG1921**
Created filter component to render on filter button click

### CODE REVIEW

File: src/components/FilterComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {View} from 'react-native'
+import BouncyCheckbox from 'react-native-bouncy-checkbox'
+import {Divider} from '@react-native-material/core'
+import json from '../data/categories-filters.json'
+import ScrollViewContainer from './ScrollViewContainer'
+
+const FilterComponent = () => {
+  const filters: string[] = json.filters
+  console.log(filters)
```
Comment (ravermz) [inline src/components/FilterComponent.tsx:10]: please remove console.log

File: src/components/ScrollViewContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import {SafeAreaView, ScrollView} from 'react-native'
+
+const ScrollViewContainer = ({children}: any) => {
+  return (
+    <SafeAreaView>
+      <ScrollView style={{margin: 16}}>{children}</ScrollView>
```
Comment (ravermz) [inline src/components/ScrollViewContainer.tsx:7]: use stylesheet

File: src/components/ViewContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import {SafeAreaView, View} from 'react-native'
+
+const ViewContainer = ({children}: any) => {
+  return (
+    <SafeAreaView>
+      <View style={{margin: 16}}>{children}</View>
```
Comment (ravermz) [inline src/components/ViewContainer.tsx:7]: use stylesheet

**Comment by RodrigoRG1921** [comment]
I already resolved them sir

**Pull request merged**